Formula/bullet.rb

```diff
@@ -23,15 +23,44 @@ class Bullet < Formula
   deprecated_option "double-precision" => "with-double-precision"
 
   depends_on "cmake" => :build
+  depends_on "enet" => :recommended
+  depends_on :python => :recommended
+  depends_on "numpy" => :recommended
 
   def install
+    dylib = OS.mac? ? "dylib" : "so"
```

Comment:
I guess those lines should be added to the linuxbrew fork only then? Fair enough I can make that change